lambdas/evaluator/handler.py

```diff
@@ -0,0 +1,241 @@
+import json
+import requests
+import base64
+import os
+from os.path import join, dirname
+from dotenv import load_dotenv
+
+dotenv_path = join(dirname(__file__), '.env')
+load_dotenv(dotenv_path)
+
+from openai import OpenAI
+chat_client = OpenAI()
+
+
+github_rest_api_headers = {'Authorization-Type': "Bearer " + os.environ['GITHUB_TOKEN'], "X-GitHub-Api-Version": "2022-11-28"}
+
+def evaluate_candidate(event, context):
+    # body = json.loads(event.get("body"))
```

Comment:
Do we still need this?
If we need, can we add TODO comment as to when we can start using them?